consumer/pom.xml

```diff
@@ -400,10 +406,45 @@
             <artifactId>unleash-client-java</artifactId>
         </dependency>
 
+
+        <dependency>
+            <groupId>com.expediagroup</groupId>
+            <artifactId>graphql-kotlin-maven-plugin</artifactId>
+            <version>${graphql-kotlin.version}</version>
+        </dependency>
     </dependencies>
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>com.expediagroup</groupId>
+                <artifactId>graphql-kotlin-maven-plugin</artifactId>
+                <version>${graphql-kotlin.version}</version>
+                <executions>
+                    <execution>
+                        <id>download schema from github</id>
+                        <goals>
+                            <goal>download-sdl</goal>
+                        </goals>
+                        <configuration>
+                            <endpoint>https://navikt.github.io/pdl/pdl-api-sdl.graphqls</endpoint>
+                            <outputDirectory>src/main/resources/pdl</outputDirectory>
+                        </configuration>
+                    </execution>
+                    <execution>
```

Comment:
genererer kotlin-klasser for representerer query-objektene som den finner i `/resources/pdl/queries`

Per nå legges klassene inn i `src/.../pdl/generated` men by-default så havner de egentlig i `target` mappen.
Fordelen med å ha de i `/src` er at det er letter å finne frem til de og de blir med i github-historikken.
Men ulempen er da at det er kode som ligger i repo som det ser ut som vi kan gjøre endringer på selv (noe vi ikke kan/bør).